comgr/PKGBUILD

```diff
@@ -23,7 +23,8 @@ build() {
   cmake -DCMAKE_INSTALL_PREFIX=/opt/rocm \
         -DCMAKE_PREFIX_PATH="/opt/rocm/llvm;/opt/rocm" \
         "$_dirname/lib/comgr"
-  make
+
+  make -j $(nproc)
```

Comment:
The number of processors should be determined by the user not the PKGBUILD, please change this back to just `make`.
See: https://wiki.archlinux.org/index.php/Makepkg#Parallel_compilation
If you would like to use all processes when building your packages please edit your own `/etc/makepkg.conf` file on your system.

Reply:
Oh I did not know about this. I'll fix all PKGBUILD. Thanks for the link.